### ISSUE TITLE
Office Addin not showing up in Office Online, but ok on Desktop

### ISSUE THREAD

**Issue by robertmuehsig** [OfficeDev/office-js#3081]
# Provide required information needed to triage your issue

## Your Environment
* Platform: Office on the web
* Host: Excel, Word, PowerPoint
* Office version number: Office online
* Operating System: Windows
* Browser (if using Office on the web): Edge on Windows 11 & 10
 
## Expected behavior
We have created a addin, which should be available on all Office platforms (Windows/Office Online etc.)

## Current behavior
The addin is deployed via the Office 365 central deployment and available for the entire organization, but (and this is very weird) it only shows up on Windows, but is not visible on Office on the web:
(same user and the deployment of the app was in December 2022)

![image](https://user-images.githubusercontent.com/756703/211786466-ff9703c0-0a15-4525-bb01-27789450a7f1.png)

## Steps to reproduce

Deploy this manifest:
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bt="http://schemas.microsoft.com/office/officeappbasictypes/1.0" xmlns:ov="http://schemas.microsoft.com/office/taskpaneappversionoverrides" xsi:type="TaskPaneApp">
  <Id>2763e7d0-66b1-49f1-b206-cd46efaca7ad</Id>
  <Version>1.0.0.1</Version>
  <ProviderName>PrimeSoft</ProviderName>
  <DefaultLocale>en-US</DefaultLocale>
  <DisplayName DefaultValue="primedocs"/>
  <Description DefaultValue="Fasten your seatbelts and get ready for a next level experience."/>
  <IconUrl DefaultValue="https://primesoft.primedocs.io/app/web/icon-40x40.png"/>
  <HighResolutionIconUrl DefaultValue="https://primesoft.primedocs.io/app/web/icon-76x76.png"/>
  <SupportUrl DefaultValue="https://www.primedocs.io"/>
  <Hosts>
    <Host Name="Presentation"/>
    <Host Name="Document"/>
    <Host Name="Workbook" />
  </Hosts>
  <DefaultSettings>
    <SourceLocation DefaultValue="https://primesoft.primedocs.io/app/web/taskpane.html"/>
  </DefaultSettings>
  <Permissions>ReadWriteDocument</Permissions>
  <VersionOverrides xmlns="http://schemas.microsoft.com/office/taskpaneappversionoverrides" xsi:type="VersionOverridesV1_0">
    <Hosts>
      <Host xsi:type="Presentation">
        <DesktopFormFactor>
            <GetStarted>
                <Title resid="GetStarted.Title"/>
                <Description resid="GetStarted.Description"/>
                <LearnMoreUrl resid="GetStarted.LearnMoreUrl"/>
            </GetStarted>
            <FunctionFile resid="Commands.Url"/>
            <ExtensionPoint xsi:type="PrimaryCommandSurface">
                <OfficeTab id="TabHome">
                    <Group id="CommandsGroup">
                        <Label resid="CommandsGroup.Label"/>
                        <Icon>
                            <bt:Image size="16" resid="Icon.16x16"/>
                            <bt:Image size="32" resid="Icon.32x32"/>
                            <bt:Image size="80" resid="Icon.80x80"/>
                        </Icon>
                        <Control xsi:type="Button" id="TaskpaneButton">
                            <Label resid="TaskpaneButton.Label"/>
                            <Supertip>
                                <Title resid="TaskpaneButton.Label"/>
                                <Description resid="TaskpaneButton.Tooltip"/>
                            </Supertip>
                            <Icon>
                                <bt:Image size="16" resid="Icon.16x16"/>
                                <bt:Image size="32" resid="Icon.32x32"/>
                                <bt:Image size="80" resid="Icon.80x80"/>
                            </Icon>
                            <Action xsi:type="ShowTaskpane">
                                <TaskpaneId>ButtonId1</TaskpaneId>
                                <SourceLocation resid="Taskpane.Url"/>
                            </Action>
                        </Control>
                    </Group>
                </OfficeTab>
            </ExtensionPoint>
        </DesktopFormFactor>
    </Host>
    <Host xsi:type="Document">
        <DesktopFormFactor>
            <GetStarted>
                <Title resid="GetStarted.Title"/>
                <Description resid="GetStarted.Description"/>
                <LearnMoreUrl resid="GetStarted.LearnMoreUrl"/>
            </GetStarted>
            <FunctionFile resid="Commands.Url"/>
            <ExtensionPoint xsi:type="PrimaryCommandSurface">
                <OfficeTab id="TabHome">
                    <Group id="CommandsGroup">
                        <Label resid="CommandsGroup.Label"/>
                        <Icon>
                            <bt:Image size="16" resid="Icon.16x16"/>
                            <bt:Image size="32" resid="Icon.32x32"/>
                            <bt:Image size="80" resid="Icon.80x80"/>
                        </Icon>
                        <Control xsi:type="Button" id="TaskpaneButton">
                            <Label resid="TaskpaneButton.Label"/>
                            <Supertip>
                                <Title resid="TaskpaneButton.Label"/>
                                <Description resid="TaskpaneButton.Tooltip"/>
                            </Supertip>
                            <Icon>
                                <bt:Image size="16" resid="Icon.16x16"/>
                                <bt:Image size="32" resid="Icon.32x32"/>
                                <bt:Image size="80" resid="Icon.80x80"/>
                            </Icon>
                            <Action xsi:type="ShowTaskpane">
                                <TaskpaneId>ButtonId1</TaskpaneId>
                                <SourceLocation resid="Taskpane.Url"/>
                            </Action>
                        </Control>
                    </Group>
                </OfficeTab>
            </ExtensionPoint>
        </DesktopFormFactor>
    </Host>
    <Host xsi:type="Workbook">
        <DesktopFormFactor>
            <GetStarted>
                <Title resid="GetStarted.Title"/>
                <Description resid="GetStarted.Description"/>
                <LearnMoreUrl resid="GetStarted.LearnMoreUrl"/>
            </GetStarted>
            <FunctionFile resid="Commands.Url"/>
            <ExtensionPoint xsi:type="PrimaryCommandSurface">
                <OfficeTab id="TabHome">
                    <Group id="CommandsGroup">
                        <Label resid="CommandsGroup.Label"/>
                        <Icon>
                            <bt:Image size="16" resid="Icon.16x16"/>
                            <bt:Image size="32" resid="Icon.32x32"/>
                            <bt:Image size="80" resid="Icon.80x80"/>
                        </Icon>
                        <Control xsi:type="Button" id="TaskpaneButton">
                            <Label resid="TaskpaneButton.Label"/>
                            <Supertip>
                                <Title resid="TaskpaneButton.Label"/>
                                <Description resid="TaskpaneButton.Tooltip"/>
                            </Supertip>
                            <Icon>
                                <bt:Image size="16" resid="Icon.16x16"/>
                                <bt:Image size="32" resid="Icon.32x32"/>
                                <bt:Image size="80" resid="Icon.80x80"/>
                            </Icon>
                            <Action xsi:type="ShowTaskpane">
                                <TaskpaneId>ButtonId1</TaskpaneId>
                                <SourceLocation resid="Taskpane.Url"/>
                            </Action>
                        </Control>
                    </Group>
                </OfficeTab>
            </ExtensionPoint>
        </DesktopFormFactor>
      </Host>
    </Hosts>

    <!-- You can use resources across hosts and form factors. -->
    <Resources>
      <bt:Images>
          <bt:Image id="Icon.16x16" DefaultValue="https://primesoft.primedocs.io/app/web/favicon-16x16.png"/>
          <bt:Image id="Icon.32x32" DefaultValue="https://primesoft.primedocs.io/app/web/favicon-32x32.png"/>
          <bt:Image id="Icon.80x80" DefaultValue="https://primesoft.primedocs.io/app/web/favicon-96x96.png"/>
      </bt:Images>
      <bt:Urls>
          <bt:Url id="Commands.Url" DefaultValue="https://primesoft.primedocs.io/app/web/commands.html"/>
          <bt:Url id="Taskpane.Url" DefaultValue="https://primesoft.primedocs.io/app/web/taskpane.html"/>
      </bt:Urls>
      <bt:ShortStrings>
          <bt:String id="GetStarted.Title" DefaultValue="primedocs"/>
          <bt:String id="CommandsGroup.Label" DefaultValue="primedocs"/>
          <bt:String id="TaskpaneButton.Label" DefaultValue="primedocs"/>
      </bt:ShortStrings>
      <bt:LongStrings>
          <bt:String id="GetStarted.Description" DefaultValue="primedocs is ready to start. Fasten your seatbelts and get ready for a next level experience."/>
          <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to open the web addin"/>
      </bt:LongStrings>
    </Resources>
  </VersionOverrides>
</OfficeApp>
```
It shows up under "Desktop Office", but is not visible in Office on the web.

# Provide additional details
I created a support request (TrackingID#2301041420000102), but the support couldn't help me and told me:
"Unfortunately, this falls out of scope of support as it deals with a 3rd party app.  Kindly respond to this email if we have your permission to archive the service request or just maybe it eventually became available after making changes?"

## Context
The other two addins are also from us and are more or less the "same" and we only changed the ids and I'm not sure if we have a issue in our manifest or why it doesn't appear in office online.

## Useful logs
- [x] Screenshots

**Comment by AlexJerabek** [comment]
Hi @robertmuehsig.

I'm sorry to hear you're having issues with your deployment. @AkhileshShah-MS, could you please investigate?

**Comment by robertmuehsig** [comment]
Ok, this was a stupid bug. The `GetStarted.LearnMoreUrl` was missing and I needed to add it to the URLs:

```
      <bt:Urls>
...
          <bt:Url id="GetStarted.LearnMoreUrl" DefaultValue="..."/>
      </bt:Urls> 
```

It seems that Office Online seems to care about these URLs more than the Desktop world. 

Unfortunately the validation after I uploaded the manifest didn't found this issue - I guess, this would be a good improvement idea.

Can be closed or can be used to track validation errors - feel free @AlexJerabek
